.github/workflows/scripts/only_latest_easystacks.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+EESSI_VERSION=${EESSI_VERSION:-"2023.06"}
+
+directory="easystacks/software.eessi.io/${EESSI_VERSION}"
+# List of example filenames
+files=($(ls "$directory"/*.yml))
```

Comment:
While maybe a rare case, but couldn't happen that one adds a new easystack file with a higher EB version first under `accel/...`? Then you would run tests for the second highest version, and would not use the new easystack file in the check.